CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+
+## [7.54.16] - 2024-08-22
+### Added
+- [Feature Preview]  Added LocationFiltersAcl capability.
```

Comment:
This is not a feature preview IMO. The group endpoint is already there and this is a supported ACL for that endpoint. This is a simple adding a missing ACL.

Feature preview are things which triggers a warning when you use them, or in the case of Toolkit, something you have to activate with a feature flag.

```suggestion
- Added new LocationFiltersAcl capability.
```